include/qthread/qthread.hpp

```diff
@@ -14,7 +14,11 @@
 template <bool> class OnlyTrue;
 template <> class OnlyTrue<true>
 {};
+#if __cplusplus >= 201103
```

Comment:
BTW, in case you were wondering, that comes from subclause 6.10.1 paragraph 4 of the C standard, and 16.1 paragraph 4 of the C\+\+ standard.  (I'm a standards guy.)